### ISSUE TITLE
SyntaxWarning

### ISSUE THREAD

**Issue by yuyaogawa** [smallworlnd/stream-lnd-htlcs#6]
When running this script, I get SyntaxWarning as follows. Then the output does not contain detail messages related to `link_fail_event` and `forward_event`.

```
~/stream-lnd-htlcs/htlc.py:25: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if self.event_outcome is 'link_fail_event':
~/stream-lnd-htlcs/htlc.py:30: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif self.event_outcome is 'forward_event':
```

When I change the syntax to `==`, the output contains as it is expected.

```
{'incoming_channel': '02ba39904141d52e730b', 'incoming_channel_capacity': 20000, 'incoming_channel_remote_balance': 14994, 'incoming_channel_local_balance': 3684, 'outgoing_channel': 'hotpotato', 'outgoing_channel_capacity': 20000, 'outgoing_channel_remote_balance': 16100, 'outgoing_channel_local_balance': 3653, 'timestamp': 1628336259, 'event_type': 'FORWARD', 'event_outcome': 'forward_event', 'event_outcome_info': {'incoming_timelock': 2063740, 'outgoing_timelock': 2063700, 'incoming_amt_msat': 1002000, 'outgoing_amt_msat': 1000000}}
```

I am not familiar with Python, but using `==` to compare objects would be appropriated?

**Comment by smallworlnd** [comment]
Yes it would be more appropriate. You're probably running a new version of python, 3.8/3.9 maybe? It's a simple fix anyway.

**Comment by smallworlnd** [comment]
Fixed in commit f995c1b8815f85c27bdedd7a94aa9278aa48543e
